d/vibed/dub.json

```diff
@@ -1,13 +1,19 @@
 {
   "name": "server",
   "dependencies": {
-    "vibe-d:http": "~>0.9.8"
+    "vibe-d:http": "~>0.10.0",
+    "vibe-d:tls": "*"
   },
   "versions": [
     "VibeManualMemoryManagement",
     "VibeHighEventPriority",
     "VibeDisableCommandLineParsing"
   ],
-  "dflags-ldc": ["-mcpu=native"],
```

Comment:
Maybe we should have the same flag everywhere. I mean if removed here, it is worth removing it for others

Reply:
It wasn't removed. It was just below) I put it back. It should stay for everyone.